admin/python/lsst/qserv/admin/configure.py

```diff
@@ -59,7 +59,7 @@
 ETC = 'etc'
 CLIENT = 'client'
 
-MYSQL = 'mysql'
+MARIADB = 'mariadb'
```

Comment:
Ok this is reasonable, but please note that mariadb is packaged as "lsst/mariadb" in eups, and this imply creation of MARIADB_DIR env var. I did this not to disrupt lsst stack behaviour.